ee/candi/cloud-providers/vcd/openapi/cluster_configuration.yaml

```diff
@@ -111,7 +111,7 @@ apiVersions:
                   example: ssd
                 template:
                   description: |
-                    Path of the template OVA that is to be used. Consists of organization (optional), catalog, template path.
+                    Path of the template OVA that is to be used. Consists of organization (optional), catalog, template path. Note: organization may not match with provider.organization (in case of shared templates).
```

Comment:
```suggestion
                    Path of the template OVA that is to be used.
                    
                    Consists of organization (optional), catalog, template path. Organization may not match with provider.organization (in case of shared templates).
```